### ISSUE TITLE
Unexpected Token

### ISSUE THREAD

**Issue by lau1944** [facundoolano/google-play-scraper#428]
.. in JSON at position 143

**Comment by TheArmKing** [comment]
### A Full Error Log (when trying to run .app):
```
    SyntaxError: Unexpected token , in JSON at position 143
    at JSON.parse (<anonymous>)
    at matches.reduce (/root/aupdate-bot/node_modules/google-play-scraper/lib/utils/scriptData.js:48:26)
    at Array.reduce (<anonymous>)
    at Object.parse (/root/aupdate-bot/node_modules/google-play-scraper/lib/utils/scriptData.js:42:18)
    at processAndRecur (/root/aupdate-bot/node_modules/google-play-scraper/lib/search.js:16:23)
    at request.then.then (/root/aupdate-bot/node_modules/google-play-scraper/lib/search.js:74:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

**Comment by icarcal** [comment]
@lau1944 and @TheArmKing  can you guys provide a code sample?

**Comment by icarcal** [comment]
Google changed the way that the data is returned to the store.
Working on a fix.

Apparently the mappings have changed too

**Comment by erishig** [comment]
see my comment in #429 for a quick fix.

**Comment by icarcal** [comment]
#430 is now up for review

**Comment by Wremeker** [comment]
Same problem :)

**Comment by lau1944** [comment]
Hope the package would update soon, I use this rest API wrapper not the package itself, so it's really tricky to edit the package file.

**Comment by yashwant-BST** [comment]
Hi, I am facing same problem. Hope that package would update soon.

**Comment by oleksandrz-tt** [comment]
Looks like issue is fixed in 8.0.2
thanks guys

**Comment by yashwant-BST** [comment]
But, the build 8.0.2 is failed.

**Comment by facundoolano** [comment]
fixed by #430

**Comment by lau1944** [comment]
Update dependency to 8.0.2, it works
